lib/swagger-explorer.ts

```diff
@@ -226,10 +232,10 @@ export class SwaggerExplorer {
   }
 
   private getOperationId(instance: object, method: Function): string {
-    if (instance.constructor) {
-      return `${instance.constructor.name}_${method.name}`;
-    }
-    return method.name;
+    return this.operationIdFactory(
+      instance.constructor?.name || '',
+      method.name
```

Comment:
This introduces a breaking change. Previously, if `instance.constructor.name` was undefined, `getOperationId` would return the `method.name`. Now, it would be `_method.name` instead

Reply:
Good catch, thanks! Will fix that on Friday 👍